src/lib/components/common/ui/tooltip.ts

```diff
@@ -0,0 +1,47 @@
+import {css, html} from 'lit';
+import {FloatElement} from '../../custom';
+import {CustomElement} from '../../injectors';
+import {property} from 'lit/decorators.js';
+
+import {hintcss} from './hintcss';
+
+@CustomElement()
+export class Tooltip extends FloatElement {
+    @property({type: String}) class = '';
```

Comment:
This isn't used anywhere -- I'd assume it's just meant to add extra classes to the tooltip?

Perhaps rename to `extraClass` or `extraClasses` with a description of where you can find the available classes to use.

Reply:
Exactly, `hintcss` provides a few classes to customize the tooltip styling, positioning and functionality.
Since there are too many to fit into a compact comment, I added a reference link for it in 7f1a30c.